lib/src/json_schema/validator.dart

```diff
@@ -63,6 +63,24 @@ class Instance {
   toString() => data.toString();
 }
 
+/// The result of validating data against a schema
+class ValidationResults {
+  ValidationResults(List<ValidationError> errors) {
+    this.errors = List.of(errors ?? []);
+  }
+
+  /// Correctness issues discovered by validation.
+  List<ValidationError> errors;
```

Comment:
I'd make this final

Reply:
Probably this is my Dart ignorance, but... if I do this, the assignment in the constructor above becomes a compilation error.